lib/workers/repository/package-files.ts

```diff
@@ -53,28 +47,30 @@ export class PackageFiles {
     let header = '';
     let removed = false;
     let truncated = false;
-    let restore: (() => void) | null = null;
+
+    const data = new Map(clone(Array.from(this.data)));
+
+    // filter all deps with skip reason
+    for (const managers of [...data.values()].filter(is.truthy)) {
+      for (const files of Object.values(managers).filter(is.truthy)) {
```

Comment:
```suggestion
    for (const manager of [...data.values()].filter(is.truthy)) {
      for (const files of Object.values(manager).filter(is.truthy)) {
```

i think manager is more accurate in this context

Reply:
nope, it's the magaers dictionary
![image](https://user-images.githubusercontent.com/1798109/185462661-404a36a0-325b-4755-ae52-bc77f3e47247.png)

Reply:
i think this is a manager with all of its detected manifest files.
`[...data.values()]` are indeed managers but when iteration over them each entry is a manger, might be wrong, im fine either way :)

Reply:
nope. those are the records of all manager per branch ;-)